Makefile

```diff
@@ -165,6 +189,15 @@ endif
 CFLAGS += -DPACKAGE_VERSION='$(PACKAGE_VERSION)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
 CPPFLAGS += -DPACKAGE_VERSION='$(PACKAGE_VERSION)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
 CFLAGS_IDO += -DPACKAGE_VERSION='$(PACKAGE_VERSION)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
+CFLAGS += -DPACKAGE_NAME='$(PACKAGE_NAME)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
+CPPFLAGS += -DPACKAGE_NAME='$(PACKAGE_NAME)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
+CFLAGS_IDO += -DPACKAGE_NAME='$(PACKAGE_NAME)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
+CFLAGS += -DPACKAGE_COMMIT_AUTHOR='$(PACKAGE_COMMIT_AUTHOR)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
+CPPFLAGS += -DPACKAGE_COMMIT_AUTHOR='$(PACKAGE_COMMIT_AUTHOR)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
+CFLAGS_IDO += -DPACKAGE_COMMIT_AUTHOR='$(PACKAGE_COMMIT_AUTHOR)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
+CFLAGS += -DPACKAGE_AUTHOR='$(PACKAGE_AUTHOR)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
+CPPFLAGS += -DPACKAGE_AUTHOR='$(PACKAGE_AUTHOR)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
+CFLAGS_IDO += -DPACKAGE_AUTHOR='$(PACKAGE_AUTHOR)' -DCOMPRESS_$(COMPRESSION_TYPE)=1
```

Comment:
this is confusing me a lot, why not just do ``-DPACKAGE_VERSION='$(PACKAGE_VERSION)' -DPACKAGE_NAME='$(PACKAGE_NAME)' -DPACKAGE_COMMIT_AUTHOR='$(PACKAGE_COMMIT_AUTHOR)' -DPACKAGE_AUTHOR='$(PACKAGE_AUTHOR)' -DCOMPRESS_$(COMPRESSION_TYPE)=1``?